.github/workflows/dependency-review.yaml

```diff
@@ -38,3 +38,5 @@ jobs:
           fail-on-severity: moderate
           allow-licenses: 0BSD, Apache-2.0, BlueOak-1.0.0, BSD-2-Clause, BSD-2-Clause-FreeBSD, BSD-3-Clause, CC-BY-3.0, CC-BY-4.0, CC0-1.0, ISC, LGPL-2.1, MIT, MIT-0, MPL-2.0, ODC-By-1.0, OFL-1.1, Python-2.0, Unicode-DFS-2016, Unlicense, Zlib, (MIT OR Apache-2.0) AND Unicode-DFS-2016, Apache-2.0 AND BSD-3-Clause, ISC AND MIT, MIT AND Zlib, MIT AND BSD-3-Clause
           allow-ghsas: ${{ inputs.allow-ghsas }}
+          allow-dependencies-licenses:
+            - 'pkg:npm/%40inkeep/widgets'
```

Comment:
Instead of hardcoding this, can we make this a variable that can be passed in via the calling workflow?

Reply:
Updated to an input

Reply:
Inputs have to be defined, so I added that for you. Thanks!